src/components/layout/organize/TabbedLayout.tsx

```diff
@@ -22,41 +20,38 @@ const useStyles = makeStyles((theme) => ({
     },
 }));
 
-interface OrganizeTabbedLayoutProps {
+interface TabbedLayoutProps {
     fixedHeight?: boolean;
+    title?: string;
+    subtitle?: string;
+    baseHref: string;
+    tabs: { defaultTab?:boolean; href: string; label: string}[];
```

Comment:
The downside with the `defaultTab` boolean is that it could be true for multiple tabs, and behavior is then not clear (although I can see by reading the code what would happen). 

A better design IMO is having some sort of unique key for the tabs (e.g. the `label` or the `href` can be considered that key) and then a separate `defaultTab` property where the value should be the key of that tab.